src/platform_impl/windows/event_loop.rs

```diff
@@ -1901,7 +1901,7 @@ unsafe fn public_window_callback_inner<T: 'static>(
         }
 
         WM_NCACTIVATE => {
-            let is_active = wparam == 1;
+            let is_active = wparam != 0;
```

Comment:
Interesting find, seems to emit `0x20_0000` for some reason (undocumented) - might be some combination of active & minimized. Overall the message behavior seems a bit unintuitive of Windows in this case ..

```suggestion
            let is_active = wparam != false.into();
```